insights/combiners/redhat_release.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, uname, rh_rel):
             self.major = uname.redhat_release.major
             self.minor = uname.redhat_release.minor
             self.rhel = '{0}.{1}'.format(self.major, self.minor)
-        elif rh_rel and rh_rel.is_rhel:
+        elif rh_rel:
```

Comment:
Hi @PaulWay - Other than making an MR to the GSS rules, it would be more suitable to keep this `RedHatRelease` combiner for RHEL only, As this combiner is designed for RHEL system only, See the docstring in L40

Reply:
I agree with Xiangce, I wanted to say something about this previously but forgot. I feel like detection for centos, fedora, oracle, etc should be done in the os-release parser not the redha-release parser/combiner. Since os-release is used by every distro, and provides a lot of information that the redhat-release file doesn't provide I feel it's better fitted.